Online/Matchmaking/SteamMatchmakingManager.cs

```diff
@@ -120,7 +121,24 @@ public override void CreateLobby(LobbyVisibility visibility, string gameMode)
 
         public override void JoinLobby(LobbyInfo lobby)
         {
-            m_JoinLobbyCall.Set(SteamMatchmaking.JoinLobby(lobby.id));
+            if (lobby.mode == "Story")
+            {
+                if (lobby.playerCount > 1)
```

Comment:
The limit should be set from a UI element, not hardcoded

Reply:
yeah you are right, i do want to attempt making the limit configurable, but for now it's hardcoded for the meantime (until i figure out making it configurable)